getting_started/data_backup.md

```diff
@@ -0,0 +1,220 @@
+
+## Automated Backups
+
+A generic cron job to run backups is provided below, to use it do the following:
+
+1. Configure [rclone](#transfers-to-cloud-storage-rclone) for your cloud storage provider on Arjuna.
+2. Create a file `~/backup.sh` with the [indicated contents](#contents-of-backupsh). Remember to replace `remote` with the name of your remote
+3. Run `crontab -e`
+4. In the editor that opens, add the following line to the bottom of the file:
+   `0 0 1 * * bash ~/backup.sh`
+5. Trigger the first backup manually: `bash ~/backup.sh`
+6. The next back will occur at 0:00 on the first day of the month.
+   *CHECK THAT THIS IS HAPPENS!*
+
+> Users are responsible for backing up their data, verifying their backups,
+> and the operation of any backup scripts. *INCLUDING THIS ONE*!
+
+### Contents of `~/backup.sh`
+
+```bash
+#!/bin/bash
+# A script to back up user folders on Arjuna
+# For more information vist:
+# https://arjunacluster.github.io/ArjunaUsers/getting_started/lindata_backup.html
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# Replace "remote" with the name of your remote (i.e. gdrive_cmu)
+# Replace "arjuna_backup" with the name of the folder you want to backup to on
+# your remote
+# Add additional filters (See https://rclone.org/filtering/) as needed
+# NOTE: Be sure to "quote" your filters and append each line with "\"
+#
+# To "Dry Run" syncing (And quickly check what will be synced), replace the
+# first line with: `rclone sync -P --dry-run remote:arjuna_backup \`
+rclone sync -P ~ remote:arjuna_backup --dry-run \
+    --ignore-case \
+    --filter "- /.cache/**" \
+    --filter "- /.bash_history" \
+    --filter "- .git/**" \
+    --filter "- .spack-env/**" \
+    --filter "+ .spack/*.yml" \
+    --filter "- .spack/**" \
+    --filter "- __pycache__/**" \
+    --filter "- *.py[cod]" \
+    --filter "- .ipynb_checkpoints/**" \
+    --filter "+ .julia/config/**" \
+    --filter "+ .julia/prefs/**" \
+    --filter "+ .julia/environments/**" \
+    --filter "- .julia/**" \
+    --filter "- .vscode-server/**" \
+    --filter "+ *" \
+
+```
+
+This will sync most files in your home directory (`~`) to `remote`.
+To sync other files, add or remove filters as needed.
+See [rclone filtering](https://rclone.org/filtering/) for more information on
+filters and how to use them.
+
+### Restrictions on Backups
+
+Please be cognizant of other users when configuring your backup scripts. Daily
```

Comment:
they put?

Reply:
(next line)

Reply:
Leaving as "they" could be "other users"